### PR TITLE
CCC 11/30/2020 demo day post

### PR DESCRIPTION
**Pull request by jdebacker**
This PR adds a blog post for the 11/30/2020 demo day on using the CCC web application.

### CODE REVIEW

**Comment by jdebacker** [comment]
@MaxGhenis a couple questions:
1. Are spaces ok in the tags?  e.g., `[corporate income tax]`
2. Do I need commas or anything separating tags?
3. Can I have a hyperlink in the author name?

**Comment by MaxGhenis** [comment]
Thanks for the post, @jdebacker. For some reason this one isn't working right when I preview it via `make server` (screenshots below - wrong title on main page and bad rendering in the post). I'll debug more tomorrow. Responses to your questions inline.

> 1. Are spaces ok in the tags?  e.g., `[corporate income tax]`

They work, but the URLs look kind of funky, e.g. http://blog.pslmodels.org/categories/#demo-days would be http://blog.pslmodels.org/categories/#demo%20days, and since posts have tags in their URL it'd be http://blog.pslmodels.org/demo%20days/2020/11/23/tax-cruncher.html instead of http://blog.pslmodels.org/demo-days/2020/11/23/tax-cruncher.html. So I think it'd be cleaner to use dashes in all tags.

> 2. Do I need commas or anything separating tags?

Per the [Fastpages docs](https://github.com/fastai/fastpages#categories), the format is `[tag1, tag2]`. I submitted a commit to your branch fixing this.

> 3. Can I have a hyperlink in the author name?

I tried this with Markdown and it didn't work. Author pages are available with other Jekyll themes, so we could explore those down the road.

Main page:
image
Post:
image

**Comment by MaxGhenis** [comment]
I asked about this on the forum: https://forums.fast.ai/t/blog-post-not-rendering-correctly/82922

**Comment by jdebacker** [comment]
@MaxGhenis I just removed a space, which could have been the issue.  But how do I preview what the post will look like?  I can view the markdown preview, but not the rendered html in my texts editor.

**Comment by MaxGhenis** [comment]
Removing the space worked!

You can test locally by cloning the repo and running `make server` from the root directory.

This post revealed that URLs include the full set of categories, which I reported in #17. If others agree with changing that, I think we should fix it before merging this in so this post has a stable URL.

**Comment by jdebacker** [comment]
@MaxGhenis Let me know if I should make any adjustment to the file name given what you decide on the URLs.

File: _posts/2020-11-30-ccc.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+I begin by illustrating the various parameters available for the user to manipulate.  These include parameters of the business and individual income tax systems, as well as parameters representing economic assumptions (e.g., inflation rates and nominal interest rates) and parameters dictating financial and accounting policy (e.g., the fraction of financing using debt).  Note that all default values for tax policy parameters represent the "baseline policy", which is defined as the current law policy in the year being analyzed (which itself is a parameter the user can change).  Other parameters are estimated using historical data following the methodology of [CBO (2014)](https://www.cbo.gov/sites/default/files/113th-congress-2013-2014/reports/49817-taxingcapitalincome0.pdf)
```
Comment (MaxGhenis) [inline _posts/2020-11-30-ccc.md:16]: missing period at end

**Review by MaxGhenis — COMMENTED** [review]
Thanks @jdebacker, let's go with `2020-12-03-demo-day-3-ccc.md` or `2020-12-03-demo-day-3-cost-of-capital-calculator.md` for more descriptiveness, up to you.

File: _posts/2020-11-30-ccc.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+ toc: true
+ layout: post
+ description: Computing the impact of taxes on business investment incentives under alternative policy scenarios.
+ categories: [demo-days, cost-of-capital-calculator, business-taxation, corporate-income-tax]
+ author: Jason DeBacker
+ title: "Demo Day 11/30: Cost-of-Capital-Calculator Web Application"
```
Comment (MaxGhenis) [inline _posts/2020-11-30-ccc.md:7]:
```suggestion
 title: "Demo Day 3: Cost-of-Capital-Calculator Web Application"
```

**Comment by jdebacker** [comment]
@MaxGhenis I believe I've made all the changes you suggested.  

I ran `make server` locally and generated `index.html` and things looked ok (though buried in a bunch of folders named after the tags).

**Pull request merged**